src/XrdOuc/XrdOucPList.hh

```diff
@@ -31,7 +31,7 @@
 /******************************************************************************/
 
 #include <cstdio>
-#include <strings.h>
+#include <cstring>
```

Comment:
Out of curiosity, why the selective replacement of "string.h" with ctring? There are still a lot of file that have "string.h" in them.

Reply:
Note that it's `strings.h`, not `string.h`. The first header is deprecated and does not contain the definitions we want. In any case, if you run the scritpt `check-headers.sh` after installing all our `*-devel` RPMs, you will see the errors these commits are fixing.